src/components/Payment/BraintreeDropInInterface.vue

```diff
@@ -83,6 +83,37 @@ export default {
 			updatingPaymentWrapper: false,
 		};
 	},
+	computed: {
+		formattedAmount() {
+			return numeral(this.amount).format('0.00');
+		},
+		applePaymentRequest() {
+			return {
+				total: {
+					label: 'Kiva',
+					amount: this.formattedAmount,
+				},
+				requiredBillingContactFields: ['postalAddress']
+			};
+		},
+		googleTransactionInfo() {
+			return {
+				totalPriceStatus: 'FINAL',
+				totalPrice: this.formattedAmount,
+				currencyCode: 'USD',
+			};
+		}
+	},
+	watch: {
+		amount(next, prev) {
+			if (next !== prev) {
+				this.btDropinInstance.updateConfiguration('paypal', 'amount', this.formattedAmount);
```

Comment:
I verified that with this config update sequence in place the amount value that Paypal is tracking is indeed updated after a donation change resulting in a basket total amount change. Once logged into the Paypal popup you can see the amount notated under the credit option, close the popup adjust the basket then re-open Paypal and see the new amount reflected.